ui/js/actions/app.js

```diff
@@ -84,6 +84,15 @@ export function doHistoryBack() {
   };
 }
 
+export function doHistoryForward() {
+  return function(dispatch, getState) {
+    //if (!history.state) return;
+    //if (history.state.index === 0) return;
+
```

Comment:
As hackrush says, this is to prevent a back button causing a blank screen when there is no history.